src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTests.vb

```diff
@@ -7819,8 +7868,65 @@ DERIVED: Public Overridable Sub f(x As Integer, Optional y As String = "")
 ]]>)
         End Sub
 
+        <Fact()>
+        Public Sub OverridingFunctionsOverloadedOnOptionalParameters_B()
+            Dim useOpts = _ImplicitDefaultOptionalParameter_
+            If Not Syntax.InternalSyntax.Parser.CheckFeatures(Syntax.InternalSyntax.Feature.ImplicitDefaultValueOnOptionalParameter, useOpts) Then
+                Assert.True(False, $"Feature{NameOf(Syntax.InternalSyntax.Feature.ImplicitDefaultValueOnOptionalParameter)} is not present")
+            End If
+            CompileAndVerify(
+<compilation>
+    <file name="a.vb">
+Imports System
+Class base
+    Public Overridable Sub f(x As Integer)
+        Console.WriteLine("BASE: Public Overridable Sub f(x As Integer)")
+    End Sub
+    Public Overridable Sub f(x As Integer, Optional y As Integer)
+        Console.WriteLine("BASE: Public Overridable Sub f(x As Integer, Optional y As Integer)")
+    End Sub
+    Public Overridable Sub f(x As Integer, Optional y As String = "")
+        Console.WriteLine("BASE: Public Overridable Sub f(x As Integer, Optional y As String = """")")
+    End Sub
+End Class
+
+Class derived
+    Inherits base
+    Public Overrides Sub f(x As Integer)
+        Console.WriteLine("DERIVED: Public Overridable Sub f(x As Integer)")
+    End Sub
+    Public Overrides Sub f(x As Integer, Optional y As Integer)
```

Comment:
>f(x As Integer, Optional y As Integer) [](start = 25, length = 38)

It would be good to have an additional test for scenarios when only base, or only derived class has explicit default value.